person.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'capitalizedecorator'
+require_relative 'trimmerdecorator'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # rubocop:disable Naming/PredicateName
+  def is_of_age?
+    age >= 18
+  end
+  # rubocop:enable Naming/PredicateName
```

Comment:
- [ ] I noticed that you disabled **rubocop** . Please remove the **rubocop disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: